### ISSUE TITLE
fact "packetbeat_version" resolved to null and will not be added

### ISSUE THREAD

**Issue by tullis** [noris-network/norisnetwork-packetbeat#5]
Hi, firstly thanks for this module.

I am having a problem compiling the module in order to begin the installation.

It looks to me like there is a chicken and egg type of problem. i.e. the module won't compile unless the command `packetbeat version` returns a value. Here is a fragment from my debug log.

```
Debug: Loading facts from /opt/puppetlabs/puppet/cache/lib/facter/packetbeat_version.rb
Info: Facter: loading custom facts from /opt/puppetlabs/puppet/cache/lib/facter/packetbeat_version.rb.
Debug: Facter: fact "packetbeat_version" resolved to null and will not be added.
Error: Could not retrieve catalog from remote server: Error 500 on SERVER: Server Error: Evaluation Error: Error while evaluating a Resource Statement, Evaluation Error: Error while evaluating a Function Call, 'versioncmp' parameter 'a' expects a String value, got Undef
```
Environment:

Debian 10.6
Puppet 5.5.21
Module version(s): 0.2.4 from forge and latest commit (1dd77032608309560e6c57c5e7882138ee1e6176)

The `lib/facter/packetbeat_version.rb` file contains the following:
```
Facter.add(:packetbeat_version) do
  setcode do
    if Facter::Core::Execution.which('packetbeat')
      packetbeat_version = Facter::Core::Execution.execute('packetbeat version 2>&1')
      %r{packetbeat version:?\s+v?([\w\.]+)}.match(packetbeat_version)[1]
    end
  end
end
```
I see that the spec contains a default fact for `packetbeat_version`:
https://github.com/noris-network/norisnetwork-packetbeat/blob/master/spec/default_facts.yml#L9

...but I'm getting a null value for that fact and a compilation failure when trying to run that module against a new host.
Thanks again.

**Comment by icefish-creativ** [comment]
Hi @tullis 
thanks für your feedback, the problem should fix with v1.0.0

**Comment by icefish-creativ** [comment]
fixed
